boards/x86/arduino_101/arduino_101.dts

```diff
@@ -58,6 +58,15 @@
 
 &spi0 {
 	status = "ok";
+	cs-gpios = <&gpio0 24 0>;
```

Comment:
I sit possible to have more than one device at the SPI bus? If yes then what with multiple cs declaration and usage?
`#define DT_SPI_FLASH_W25QXXDV_GPIO_SPI_CS_PIN		DT_SNPS_DESIGNWARE_SPI_B0001000_CS_GPIOS_PIN`

Reply:
Its possible from a DTS point of view, however we don't handle the code generation side of this correctly today for more than a single device on the spi bus.

Reply:
got it